### DISCUSSION TITLE
ModuleNotFoundError: Module not found: Error: Can't resolve 'next/Head' in '/vercel/path0/pages'

### DISCUSSION THREAD

**Discussion by harsh3294** [vercel/vercel#6480] [Help] [answered]
ModuleNotFoundError: Module not found: Error: Can't resolve 'next/Head' in '/vercel/path0/pages'
13:35:09.725  > Build error occurred

13:35:09.726
  Error: > Build failed because of webpack errors

13:35:09.726
  at /vercel/path0/node_modules/next/dist/build/index.js:15:924

13:35:09.726
  at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:6:584)
![image](https://user-images.githubusercontent.com/57228178/126039475-b713ae0a-8187-4623-8b4d-d86f3b56aedf.png)

i tried  all the possible ways i can do that but issue didnt resolved .  

is there any way to resolve this error?
because the code is working fine in the local development

**Comment by barelyhuman** [comment, marked as answer]
Did you try changing it to, 
`import Head from 'next/head'` instead?

**Reply by harsh3294** [reply]
still same
![image](https://user-images.githubusercontent.com/57228178/126039418-1cde0d5f-cd16-47ac-9e5b-ea973caef3f6.png)

**Reply by barelyhuman** [reply]
In that case, is it possible to create a minimal reproducible example of the same? 
Either using codesandbox or a git repo, either works.

**Reply by leerob** [reply]
Agree with @barelyhuman, a small example would help out a bunch!

**Reply by harsh3294** [reply]
the code is working fine in the local development but while deploying to vercel it is giving me the error

**Reply by barelyhuman** [reply]
@harsh3294  yes, but for us to debug why it’s happening. It would be easier with an example codebase
